package/index.ts

```diff
@@ -384,10 +384,12 @@ export class UploadArea {
     if (!this.options.manualStart) this.putFilesToQueue();
   }
 
-  private putFilesToQueue(): void {
-    if (!this.fileList) return;
+  private putFilesToQueue(files?: IUploadFile[] | null): void {
```

Comment:
`| null` is not needed. Please remove it.

Reply:
Removed.